client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -96,6 +97,17 @@ export default function PaymentInfoBlock( {
 		return <PaymentInfoBlockWrapper>{ translate( 'In-App Purchase' ) }</PaymentInfoBlockWrapper>;
 	}
 
+	if ( purchase.isAutoRenewEnabled && ! hasPaymentMethod( purchase ) ) {
+		return (
+			<PaymentInfoBlockWrapper>
+				<div className={ 'manage-purchase__no-payment-method' }>
+					<Icon icon={ warning } />
+					{ translate( 'No payment method' ) }
```

Comment:
Not urgent, but we may be able to make this message more user-friendly. I am not thinking of better words, but wanted to flag it regardless!

Also, I think we may need to look at `isBackupMethodAvailable` in our checks to make sure we communicate that a backup method will be used.

Reply:
@daledupreez Can you elaborate on your second point? Do you mean something like this:
![image](https://user-images.githubusercontent.com/6851384/178250385-5c3a49a5-7381-4dd3-9eb4-71fa05807305.png)
Currently, you see this on your subscription if it has card B assigned as a payment method and card A exists with the "backup" checkbox set. If you then delete card A, you then see:

![image](https://user-images.githubusercontent.com/6851384/178250668-f9f056a3-b0a6-4a31-9755-0431964f4ed1.png)

This seems to agree with the docs for this backup checkbox:

![image](https://user-images.githubusercontent.com/6851384/178251146-b81b9928-dad1-4328-b0de-50420b9d0a88.png)
I need to check through the code to see if we still use a backup in the case where the purchase payment method has been deleted, and not just when the purchase payment method exists but fails.

Reply:
OK, it looks like we use the term "payment method" in our inline documentation, so I think my comment here was that keeping just the phrase "No payment method" is not that helpful from a user perspective. Perhaps we can consider using language more like:
`You don't have a payment method for renewals`

> I need to check through the code to see if we still use a backup in the case where the purchase payment method has been deleted, and not just when the purchase payment method exists but fails.

Given this question, we almost certainly need input from @Automattic/shilling to comment on whether we would use a backup payment method in this case (and possibly in others).

Reply:
🤔  That is a good question. I guess I would assume that if the payment method on a subscription is removed, no renewal will be attempted for that subscription, so a backup payment method would not be used. The messaging added by https://github.com/Automattic/wp-calypso/pull/58200 seems to imply that as well. However, I'm not certain of that so I'm going to ask the rest of the team.

Reply:
> I need to check through the code to see if we still use a backup in the case where the purchase payment method has been deleted, and not just when the purchase payment method exists but fails.

The conclusion seems to be that we will not use a backup payment method in this case.

Reply:
Thanks for digging into that, @sirbrillig! 🙇 

We can likely stick with wording more like:
> You don't have a payment method for renewals

Reply:
That's been changed @daledupreez, as agreed. 

![image](https://user-images.githubusercontent.com/6851384/180159179-6cd7a625-87d1-4334-8695-796bd7465873.png)